packages/webview_flutter/webview_flutter_android/android/src/main/java/io/flutter/plugins/webviewflutter/JavaScriptChannel.java

```diff
@@ -58,6 +58,13 @@ public void postMessage(final String message) {
     }
   }
 
+  // Adapter for Javascript code expecting "exec" method
```

Comment:
**issue**: Adding this here will make it so that we are unable to PR this back into the google plugin. I think we need to find a different way to make this happen.

Reply:
This may be good to discuss in standup. I'm also a bit unclear of what the plan is with the webview_flutter package now that we're adding more to it.